### DISCUSSION TITLE
New GraphQL improvements

### DISCUSSION THREAD

**Discussion by marina-chibizova** [leonsteinhaeuser/project-beta-automations#49] [Ideas]
Does [this](https://github.blog/changelog/2022-06-23-the-new-github-issues-june-23rd-update/#%F0%9F%A4%96-graphql-api-improvements) allow more automation in projects beta? Would be nice to be able to use project fields as webhooks for events, e.g:
- close issue when status is changed to `closed`
- fetch a deadline from the date field and enable notification-bot to remind about it
- change assignee when single select value is changed
- etc

It would make project flow super flexible and useful

**Comment by leonsteinhaeuser** [comment]
Thanks for sharing that resource. I have to admit that I did not notice that change. Before I can tell anything about these changes, I need to invest some time before I can improve the workflow.

**Reply by leonsteinhaeuser** [reply]
My answer might be a bit late, but the changes introduced by GitHub does not affect the event API which is necessary in order to get the event's you've asked for.

**Reply by marina-chibizova** [reply]
thanks for responding! that's a pity. what do they change? how can they be used in project automation?

**Reply by leonsteinhaeuser** [reply]
They changed the field names of the API.

Examples: 
- `projectNextItem` to `projectV2Item`
- `updateProjectNextItemField` to `updateProjectV2ItemFieldValue`
- ...

**Reply by marina-chibizova** [reply]
ok, thanks
